contracts/truefi2/LoanFactory2.sol

```diff
@@ -37,7 +37,7 @@ contract LoanFactory2 is ILoanFactory2, Initializable {
     mapping(IDebtToken => bool) public override isLoanToken;
 
     IPoolFactory public poolFactory;
-    address public lender;
+    address DEPRECATED__lender;
```

Comment:
```suggestion
    address private DEPRECATED__lender;
```

Reply:
isn't it the same thing?

Reply:
Ok, it's not. They are internal by default